cabby/data/metagraph/utils.py

```diff
@@ -169,75 +144,81 @@ def convert_multidi_to_weighted_undir_graph(
   Returns:
     out_graph: graph with weighted undirected edges.
   """
+  edges = collections.defaultdict(dict)
+  for node, neighbor_dict in in_graph.adjacency():
+    for neighbor, edges_dict in neighbor_dict.items():
+      id_pair = tuple(sorted([node, neighbor]))
+      if 'weight' not in edges[id_pair]:
+        edges[id_pair]['weight'] = 0.0
+      for _, edge_data in edges_dict.items():
+        if 'true_length' in edge_data:
+          weight = edge_data['true_length']
```

Comment:
It would be good to include a brief comment as to why true_length vs length exists, perhaps with a pointer to the Map code that does this.

Reply:
Done in e2087a6